extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessState.scala

```diff
@@ -51,6 +52,7 @@ object ProcessActionType extends Enumeration {
 }
 
 object StateStatus {
+  type StateId = String
```

Comment:
Maybe Status, not state. We use "state" for naming status + additional information like allowed actions, deployed id, and version. Also if we don't change the method `name` name, maybe we should also use "name" here to avoid confusion.

Reply:
Fixed (but this is a bit confusing, I could have introduced not intentionally some more misleading names).